src/cobertura.rs

```diff
@@ -232,14 +232,13 @@ impl ToString for ConditionType {
 
 fn get_coverage(
     results: CovResultIter,
+    sources: Vec<String>,
     demangle: bool,
     demangle_options: DemangleOptions,
 ) -> Coverage {
-    let sources = vec![".".to_owned()];
     let packages: Vec<Package> = results
         .map(|(_, rel_path, result)| {
             let all_lines: Vec<u32> = result.lines.iter().map(|(k, _)| k).cloned().collect();
-
```

Comment:
Nit: please reinstate the newline, as it is an unrelated change

Reply:
No problem. Done.